zarr/storage.py

```diff
@@ -1879,6 +1884,157 @@ def __delitem__(self, key):
             self._invalidate_value(key)
 
 
+class ABSStore(MutableMapping):
+    """Storage class using Azure Blob Storage (ABS).
+
+    Parameters
+    ----------
+    container_name : string
+        The name of the ABS container to use.
+    prefix : string
+        Location of the "directory" to use as the root of the storage hierarchy
+        within the container.
+    account_name : string
+        The Azure blob storage account name.
+    account_key : string
+        The Azure blob storage account acess key.
+    blob_service_kwargs : dictionary
+        Extra arguments to be passed into the azure blob client, for e.g. when
+        using the emulator, pass in blob_service_kwargs={'is_emulated': True}
+
+    Notes
+    -----
+    In order to use this store, you must install the Azure Blob Storage
+    https://github.com/Azure/azure-storage-python/tree/master/azure-storage-blob_ version >= 1.3.0.
```

Comment:
Not sure this link will format well as RST? Should add the class into the API docs then check RTFD.

Reply:
True, probably won't format well with RST. Sorry what does this mean exactly: "Should add the class into the API docs then check RTFD."?

Reply:
what about his @alimanfoo